### PR TITLE
Skip tests on release tag

### PR DESCRIPTION
**Pull request by adriancole**
Save overhead by not re-testing the same commit that was already
tested on master. This matches travis behaviour.

### CODE REVIEW

**Comment by adriancole** [comment]
goofed something

Screen Shot 2020-11-17 at 12 46 50 PM

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -10,25 +12,22 @@ on:
...
-  test:
-    runs-on: ubuntu-20.04 # aka focal, because this is what we use in Travis
+  build:
+    runs-on: ubuntu-20.04  # aka focal, because this is what we use in Travis
     steps:
       - name: Checkout Repository
         uses: actions/checkout@v2
         with:
           fetch-depth: 10
-      # Remove apt repos that are known to break from time to time.
```
Comment (adriancole) [inline .github/workflows/ci.yml:20]: relic as we don't install any packages

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -10,25 +12,22 @@ on:
...
 jobs:
-  test:
-    runs-on: ubuntu-20.04 # aka focal, because this is what we use in Travis
+  build:
+    runs-on: ubuntu-20.04  # aka focal, because this is what we use in Travis
     steps:
       - name: Checkout Repository
         uses: actions/checkout@v2
         with:
           fetch-depth: 10
```
Comment (anuraaga) [inline .github/workflows/ci.yml:21]: Do we need `fetch-depth: 0` maybe for license plugin?

**Review by anuraaga — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 2 above)
Comment (adriancole) [reply .github/workflows/ci.yml:21]: this one doesn't use the license plugin, but it is a good point. we should do that for copy/pasta defense

**Pull request merged**